Agriculture Project/MachineLearning_algo/CropRecommendationTesting.py

```diff
@@ -0,0 +1,502 @@
+import pandas as pd
+import pickle
+import numpy as np
+
+class CropRecommendationTesting:
+    def __init__(self,Saved_model_name):
+        self.Saved_model_name = Saved_model_name
+
+    def test(self):
+        model = pickle.load(open(self.Saved_model_name, 'rb'))
+
+        #testing for apple class
+        print('Apple class testing ......')
+
+        # test case 1
+        test_case1 = [20,134,199,22,92,5,112]   
+        result = model.predict(np.array(test_case1).reshape(1,7))
+        if result[0] == 'apple': print('Test case 1 sucess')
+        else:print('Test case 1 Fail')
+
+        # test case 2
+        test_case2 = [0,20,195,21,90,5,100]   
+        result = model.predict(np.array(test_case2).reshape(1,7))
+        if result[0] == 'apple': print('Test case 2 sucess')
+        else:print('Test case 2 Fail')
+
```

Comment:
Follow this guide on writing test cases using unit-test
 https://realpython.com/python-testing/

Change this entire script